Assets/AnkrSDKImporter/Editor/AnkrSDKImporter.cs

```diff
@@ -0,0 +1,205 @@
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using SimpleJSON;
+using UnityEditor;
+using UnityEngine;
+using UnityEditor.PackageManager.Requests;
+using UnityEditor.PackageManager;
+using PackageInfo = UnityEditor.PackageManager.PackageInfo;
+
+namespace AnkrSDKImporter.Editor
+{
+   [InitializeOnLoad]
+   public class AnkrSDKImporter
+   {
+      private const string CompanyNameToRejectImport = "Ankr";
+      private const string ProductNameToRejectImport = "Ankr SDK";
+
+      private static readonly List<PackageData> PackagesToTryToImport = new List<PackageData>()
+      {
+         new PackageData("com.cysharp.unitask", "2.3.1", external:false), 
```

Comment:
Shouldn't we use UnitaskRegistryScope instead of hardcoded string?

Reply:
The idea here was that the registry scopes are different abstractions with the same value, but we can substitute it here for sure